docs/site/.helm/templates/15-registry-scanner.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: registry-scanner
+  annotations:
+    "werf.io/replicas-on-creation": "1"
+  labels:
+    service: registry-scanner
```

Comment:
Sound like it is a security scanner for the registry

Reply:
Work-in-progess naming:)